tools/topology/sof-byt-codec.m4

```diff
@@ -54,10 +54,10 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 
 # PCM Media Playback pipeline 3 on PCM 1 using max 2 channels of s32le.
 # 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-pcm-media.m4,
+PIPELINE_PCM_ADD(sof/pipe-low-latency-playback.m4,
```

Comment:
Ok, so "media" is remove here, but this also removes the mixer and SRC. Easiest fix is to create a "pipe-pcm-mixer" that is basically the same as "pipe-pcm-media" minus the SRC.

Reply:
I am really not happy with the topologies. The mixer should be the start of a new pipeline, not embedded in one specific pipeline. It's been 2 years and it's probably time to rethink all this...

Reply:
Ignore my last comment, I thought mixer was in the media pipe, it's in the low latency pipe. Yep I agree, low-latency pipe needs renamed to include "mixer" and the PCM should be removed from it so it starts at the mixer.